@xen-orchestra/web-core/lib/components/card/CardTitle.vue

```diff
@@ -0,0 +1,64 @@
+<!-- v1.0 -->
+<template>
+  <div class="card-title">
+    <div class="left-blok">
+      <div class="title-and-counter">
+        <span class="typo h6-medium"><slot name="left"></slot></span>
+        <UiCounter v-if="count" class="count" :value="count" :color="counterColor" />
+      </div>
+      <span class="typo p3-regular bottom-left"><slot name="bottom-left"></slot></span>
+    </div>
+
+    <div class="right-blok">
+      <span class="typo h7-semi-bold right"> <slot name="right"></slot></span>
```

Comment:
Same here for the `v-if` check (I also changed the tag element to `<div>`, as it can contain multiple elements):
```suggestion
      <div v-if="$slots.info" class="typo h7-semi-bold info"><slot name="info" /></div>
```